galaxy/wrapper/deepTools_macros.xml

```diff
@@ -308,23 +380,25 @@ This tool is developed by the `Bioinformatics and Deep-Sequencing Unit`_ at the
     </xml>
 
     <xml name="image_file_format">
-        <param name="outFileFormat" type="select" label="Image file format">
+        <param argument="--outFileFormat" type="select" label="Image file format">
             <option value="png" selected="true">png</option>
             <option value="pdf">pdf</option>
             <option value="svg">svg</option>
             <option value="eps">eps</option>
-            <option value="emf">emf</option>
         </param>
     </xml>
 
-    <xml name="missingDataAsZero">
-                <param name="missingDataAsZero" type="boolean" truevalue="yes" falsevalue="no" checked="True"
-                    label ="Treat missing data as zero"
-                    help  ="This parameter determines if missing data should be treated as zeros. If unchecked, missing data will be ignored and not included in the output file. Missing data is defined as those regions for which both BAM files have 0 reads." />
+    <xml name="keepNAs">
+        <param argument="--keepNAs" type="boolean" truevalue="--keepNAs" falsevalue="" checked="False"
```

Comment:
@bgruening we renamed this to be --skipNonCoveredRegions (or simply --skipNAs). 

The help says:

 'This parameter determines if non covered regions regions without overlapping reads) in a bam file should 
  be skipped. The default is to treat those regions as having a value of zero.  The decision to skip non 
 covered regions depends on the interpretation of the data. Non covered regions may represent for 
 example repetitive regions that want to be skipped.